src/alacris_lib/state_update.ml

```diff
@@ -81,22 +81,23 @@ let post_operation_general_kernel : Ethereum_chain.Operation.t -> Address.t -> T
      Lwt_exn.return receipt))
 
 
-let post_operation_general : Ethereum_chain.Operation.t -> Address.t -> TokenAmount.t -> TransactionReceipt.t Lwt_exn.t =
```

Comment:
`general` is not a very useful name suffix. Is there already a `post_operation` ? If not, just use that. If yes, does it call this function? Maybe nest it, or call it `helper` if that's the previous is the only caller.

Reply:
I renamed it as "process_ethereum_operation"

Reply:
This problem was addressed.